src/edge/__init__.py

```diff
@@ -1,6 +1,6 @@
 from django.db.backends.signals import connection_created
 
-__version__ = '2.6.0'
+__version__ = '2.8.0'
```

Comment:
What's the reason for bumping the version here in the PR?  I think the flow is to run https://github.com/ginkgobioworks/edge#versioning on the master branch which will automatically bump the version and push a tag. Were you trying to release this branch on pypi?

Reply:
I think I just didn't know what I was doing.